mod-bulk-edit/src/main/resources/global/add-okapi-permissions.feature

```diff
@@ -0,0 +1,83 @@
+Feature: prepare data for api test
+
```

Comment:
Is it possible to use table adminAdditionalPermissions as common approach to add needed permissions for admin user,  but not own separate feature.